### ISSUE TITLE
Declarative backup configurations

### ISSUE THREAD

**Issue by ttrssreal** [ttrssreal/jix#14]
Restic in hm not yet implemented D:

**Comment by ttrssreal** [comment]
https://github.com/nix-community/home-manager/issues/5009

**Comment by ttrssreal** [comment]
unblocked now, rclone got merged into hm!

**Comment by ttrssreal** [comment]
done after #33 and #32
